MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -64,6 +64,7 @@ public class MediaEncoder : IMediaEncoder, IDisposable
 
         private string _ffmpegPath = string.Empty;
         private string _ffprobePath;
+        private int threads = EncodingHelper.GetNumberOfThreads(null, _configurationManager.GetEncodingOptions(), null);
```

Comment:
error CS0236: A field initializer cannot reference the non-static field, method, or property 'MediaEncoder._configurationManager'